crates/aptos/src/stake/mod.rs

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::types::{CliCommand, CliResult, CliTypedResult, TransactionOptions};
+use aptos_rest_client::Transaction;
+use aptos_transaction_builder::aptos_stdlib;
+use aptos_types::account_address::AccountAddress;
+use async_trait::async_trait;
+use clap::Parser;
+
+/// Tool for manipulating stake
+///
+#[derive(Parser)]
+pub enum StakeTool {
+    AddStake(AddStake),
+    UnlockStake(UnlockStake),
+    WithdrawStake(WithdrawStake),
+    IncreaseLockup(IncreaseLockup),
+    InitializeStakeOwner(InitializeStakeOwner),
```

Comment:
Maybe `InitializeStakePool` instead? same for `initialize-stake-pool` to be consistent with the node operation where they specify `--pool-address`

Reply:
initialize-validator also initializes stake pool. so keeping as is. But will revisit the naming with Kevin